teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +122,283 @@ requirements, and provenance attestations can be made to meet software supply ch
+Tekton Hub can indicate the source Catalog of each resource, such as
+through a badge or tag. 
+
+To add the Official Catalog, we only have to modify the Catalogs 
+[configuration][hub-config] in the Hub, as such:
+
+```yaml
+catalogs:
+  
+  # community catalog
+  - name: tekton
+    org: tektoncd
+    type: community
+    provider: github
+    url: https://github.com/tektoncd/catalog
+    revision: main
+    
+  # official catalog  
+  - name: tekton-official
+    org: tektoncd
+    type: official
+    provider: github
+    url: https://github.com/tektoncd/catalog-official
+    revision: main
+```
+
+This provides extensibility to the support tiers: we can add additional
+Catalogs for other support tiers as needed in the future. 
+
+#### Responsibilities
+
+##### Resource Ownership
+
+The responsibilities of owning a resource include but are not limited to:
+
+1. Reviewing and approving all changes to the resource.
+2. Resolving any failures of tests validating the resource.
+3. Responding to and resolving any issues reported concerning the resource.
+4. Updating the resources to be compatible with new versions of Tekton, and
+   other dependencies.
+
+##### Catalog Ownership
+
+The responsibilities of owning a Catalog include but are not limited to:
+
+1. Reviewing and approving new resources contributed to the Catalogs.
+2. Maintaining the health of the testing infrastructure.
+3. Triaging issues and involving the owners of the affected resources.
+4. Ensuring the resources meet the quality standards of the Catalog.
+
+#### Design Evaluation
+
+The Catalogs will leverage existing structure and infrastructure that 
+supports the existing `tektoncd/catalog`. Decoupling the support tiers 
+into separate repositories makes it easy to distinguish the support tiers, 
+enforce the applicable quality standards, and leverage existing structure 
+and infrastructure. Moreover, reusing the existing Catalog at 
+`tektoncd/catalog` minimizes effort to provide the support tiers.
+
+#### Alternatives
+
+##### 1. One Repository and Use Annotations
+
+Instead of creating new repository, add a new `tekton.dev/tier` annotation
+to the resource yaml file with the relevant support tier, for example:
+
+```yaml
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/tags: image-build
+    tekton.dev/tier: official
+```
+
+The `tekton.dev/tier` annotation should be optional. If the annotation 
+is not specified, `community` support tier is the default.
+
+###### Design Evaluation
+
++ This provides backward compatibility to the existing resources
+  in the Tekton Catalog, given the default to `community`.
+- It makes it difficult to distinguish between community and official 
+  resources in the one repository, without digging into the yamls.
+  It also makes it harder for maintainers to enforce the quality 
+  requirements, such as SLOs, for the tiers when they are all mixed up.
```

Comment:
I would say it depends on the interface. Via Hub / CLI it can be very easy do distinguish resources.

Reply:
that's fair, updated the design evaluation to reflect this